packages/terra-form-select/src/_FrameUtil.js

```diff
@@ -15,24 +15,24 @@ class FrameUtil {
   /**
    * Determines the dropdown position.
    * @param {Object} props - The component props.
-   * @param {Object} state - The component state.
    * @param {ReactNode} target - The select wrapper.
    * @param {ReactNode} dropdown - The dropdown.
+   * @param {number} maxHeight - The maxHeight of the dropdown dropdown.
    * @return {Object} - The calculated dropdown attributes.
    */
-  static dropdownPosition(props, state, target, dropdown) {
+  static dropdownPosition(props, target, dropdown, maxHeight) {
     const style = Object.assign({}, props).style || {};
     const { height } = dropdown.getBoundingClientRect();
     const { bottom, width, top } = target.getBoundingClientRect();
 
     const spaceBelow = window.innerHeight - bottom;
-    const maxHeight = style.maxHeight ? parseInt(style.maxHeight, 10) : Infinity;
-    const canFitBelow = maxHeight < spaceBelow || height < spaceBelow || spaceBelow > top;
+    const maximumHeight = parseInt(style.maxHeight || maxHeight, 10) || Infinity;
```

Comment:
If we are first classing the maxHeight, I think it would be better to put precedence on that first, and then check the styles object.

Reply:
I lean towards inline styles always having precedence. It follows similar expectations from CSS. Realistically it probably doesn't matter. This scenario could only be observed if a user sets a different value in both places which would be ambiguous either way. 

If there are strong preferences one way or another I'll update the precedence.